plugins/fossa/src/api/FossaClient.ts

```diff
@@ -14,28 +14,38 @@
  * limitations under the License.
  */
 
-import { DiscoveryApi } from '@backstage/core';
+import { DiscoveryApi, IdentityApi } from '@backstage/core';
 import fetch from 'cross-fetch';
 import { FindingSummary, FossaApi } from './FossaApi';
 
 export class FossaClient implements FossaApi {
   discoveryApi: DiscoveryApi;
+  identityApi?: IdentityApi;
```

Comment:
How come this one ended up being optional?

Reply:
I was probably thinking about backwards compatibility, but inconsistently it seems. Changed to required.